rpcs3/Emu/Cell/Modules/cellMouse.h

```diff
@@ -39,12 +39,12 @@ struct CellMouseInfoTablet
 
 struct CellMouseData
 {
-	u8 update;
-	u8 buttons;
-	s8 x_axis;
-	s8 y_axis;
-	s8 wheel;
-	s8 tilt;
+	be_t<u8> update;
+	be_t<u8> buttons;
+	be_t<s8> x_axis;
+	be_t<s8> y_axis;
+	be_t<s8> wheel;
+	be_t<s8> tilt;
```

Comment:
placibo change, byte long types are the same in big endian and small endian

Reply:
lol. is it a problem?

Reply:
the rest of the code in the project doesnt apply BE for these types, just for consistency.